src/main/java/org/jitsi/impl/protocol/xmpp/ChatRoomImpl.java

```diff
@@ -116,9 +116,6 @@
     private final CopyOnWriteArrayList<ChatRoomLocalUserRoleListener> localUserRoleListeners
             = new CopyOnWriteArrayList<>();
 
-    /**
-     * Nickname to member impl class map.
-     */
```

Comment:
Why remove this doc?

Reply:
Because it's wrong (the keys are the full JIDs, not just the nickname).